packages/ember-simple-auth/tests/simple-auth/setup-test.js

```diff
@@ -168,6 +168,22 @@ describe('setup', function() {
         expect(this.authorizer.authorize).to.have.been.calledThrice;
       });
 
+      it('authorizes requests going to a foreign origin if the hosts on whitelist allow subdomain to origin', function() {
+        Configuration.crossOriginWhitelist = ['http://*.other-domain.com'];
+        setup(this.container, this.application);
+        Ember.$.get('http://test.other-domain.com/path/query=string');
+
+        expect(this.authorizer.authorize).to.have.been.calledOnce;
+
+        Ember.$.get('http://another-test.other-domain.com/path/query=string');
+
+        expect(this.authorizer.authorize).to.have.been.calledTwice;
+
+        Ember.$.get('http://test2.other-domain.com/path/query=string');
+
+        expect(this.authorizer.authorize).to.have.been.calledThrice;
+      });
```

Comment:
Can you add test cases for

- `http://test2.other-domain.com:80/path/query=string` // should be called
- `http://test2.other-domain.com:1234/path/query=string` // should not be called

plus a test for when you configure `Configuration.crossOriginWhitelist = ['http://*.sub.other-domain.com:1234'];`:

- `http://test2.sub.other-domain.com/path/query=string` // should not be called
- `http://wrong.test2.other-domain.com:1234/path/query=string` // should not be called
- `http://sub.test2.other-domain.com:1234/path/query=string` // should be called